ONSdigital/florence
# Updated linter, no conflict with prettier

### What

* Eslint rules now extend prettier formatter along with react/prettier rules.
* Eslinter updated

### How to review

* Check linting and prettier don't conflict
* Ensure everything still builds as normal

1. Run npm install, to ensure the new dependencies are installed
2. Navigate to ./src
3. Run `npm run eslint-check`
4. Hopefully it should say no conflict
5. Try create a conflict in the .eslintrc file add "semi: "error""
6. Rerun step three - this time it should say there is a conflict

### Who can review

Member of the ONS DP JavaScript Guild / Frontend Guild
